.github/workflows/pull-request.yml

```
@@ -449,3 +449,31 @@
+    name: Check for Merge Conflicts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          # Use the appropriate ref based on the event type
+          ref: ${{ github.event.pull_request.head.ref }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check Mergeable Status via API
+        run: |  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)